tests/AntiMattr/Tests/MongoDB/Migrations/Tools/Console/Command/StatusCommandTest.php

```diff
@@ -438,12 +438,19 @@ public function testExecuteWithShowingVersions()
             ->with("\n <info>==</info> Available Migration Versions\n")
         ;
 
-        $this->output->expects($this->at(39))
+        // Symfony 4.2 has different output
+        $consoleVersion = $this->getSymfonyConsoleVersion();
```

Comment:
This test is not ideal but for now we can leave it like so until we have better idea.